### PR TITLE
use npm scripts

### PR DESCRIPTION
**Pull request by paulirish**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -46,20 +46,28 @@ after_success:
...
 Install the script:
 
     yarn add --dev https://github.com/ebidel/lighthouse-ci
 
-Next, in `.travis.yml` call [`runlighthouse.js`][runlighthouse-link] as the last step in `after_success`:
+Add an NPM script to your `package.json`:
+
+```js
+   "scripts": {
+     "lighthouse": "lighthouse-ci https://staging.example.com"
```
Comment (ebidel) [inline README.md]: This is probably unnecessary. Might remove it later but we can keep for now. I think it's easy enough to have users run `lighthouse-ci https://staging.example.com"`

File: README.md [hunk 2]
```diff
@@ -46,20 +46,28 @@ after_success:
...
+
+Next, in `.travis.yml` call [`yarn run lighthouse"`][runlighthouse-link] as the last step in `after_success`:
 
 ```yml
 install:
   - npm install # make sure to install the deps when Travis runs.
 after_success:
   - ./deploy.sh # TODO(you): deploy the PR changes to your staging server.
-  - node node_modules/lighthouse-ci/runlighthouse.js https://staging.example.com
+  - yarn run lighthouse
```
Comment (ebidel) [inline README.md]: Actually yea. Let's just run the binary directly? `yarn` doesn't need to get in the way.

Comment (paulirish) [reply README.md]: we can't without reaching inside `node_modules` which is not very desirable.  

i've changed it to use `npm` instead of `yarn`.

**Comment by paulirish** [comment]
dopeness. lg.

**Comment by abdonrd** [comment]
@paulirish @ebidel I have tried it, but it does not work for me. I miss something?

https://github.com/abdonrd/abdonrd.com/pull/19/files

Thanks in advance!

**Comment by paulirish** [comment]
Hmmm


```
$ yarn run lh https://abdonrd-develop.firebaseapp.com
yarn run v1.3.2
$ lighthouse-ci https://abdonrd-develop.firebaseapp.com
/bin/sh: 1: lighthouse-ci: not found
```

but yet:
https://github.com/ebidel/lighthouse-ci/blob/57a3efc66a330ba9b48ac22bf5ba61a209d565db/package.json#L8-L10

i am confused. :/

**Comment by ebidel** [comment]
Did some fixes and things seem to be working now.

https://github.com/ebidel/lighthouse-ci/commit/e57e8972be1fc364f90ae4dcdc2c5d11aa227f01
https://github.com/ebidel/lighthouse-ci/commit/14c3cbd90f411aca339ccdaf5a9230ef078ef382

**Comment by abdonrd** [comment]
Now it works, thanks!

**Pull request merged**